temporalio/test/worker_workflow_handler_test.rb

```diff
@@ -764,6 +764,60 @@ def test_update_with_start_cancel
     end
   end
 
+  def test_update_with_start_start_fail
```

Comment:
Not related to this PR, just added here because was missed in original update-with-start PR